src/lib.rs

```diff
@@ -314,6 +314,33 @@ where
         Ok(())
     }
 
+    /// Set mirror/vertical flip.
```

Comment:
I think just mentioning mirroring behaviour is fine:

```suggestion
    /// Set mirror enabled/disabled.
```